sway/ipc-json.c

```diff
@@ -1002,6 +1003,17 @@ json_object *ipc_json_describe_input(struct sway_input_device *device) {
 		}
 	}
 
+	if (device->wlr_device->type == WLR_INPUT_DEVICE_POINTER) {
+		struct input_config *ic = input_device_get_config(device);
+		if (ic != NULL) {
```

Comment:
We want to expose the `scroll_factor` field even if we don't have an input config, I think. Should default to 1.0 in this case.

Reply:
So `float scroll_factor = ic->scroll_factor`?

Reply:
I mean that we should call `json_object_object_add` even if `ic == NULL`.